packages/storybook/stories/INTERNAL/UnsupportedBrowser/UnsupportedBrowser.stories.tsx

```diff
@@ -34,16 +36,20 @@ const getDocs: () => JSX.Element = () => {
     <Stack>
       <SingleLineBetaBanner />
       <Title>Unsupported Browser</Title>
-      <Description>Component to display help information when the app is loaded in a unsuppported browser.</Description>
+      <Description>
+        Component to display help information when the app is loaded in a unsuppported environment. Shows different
+        screens based on what the environment info from the calling SDK is reporting.
+      </Description>
       <Heading>Using in a Modal</Heading>
       <Description>This component while used in a modal here can also be used as it's own page.</Description>
```

Comment:
nit:
This component can also be used as it's own page or as in a modal as shown here.